src/components/NavBar.vue

```diff
@@ -0,0 +1,42 @@
+<template lang="pug">
+  nav.navbar-container
+    img.navbar-image(src="../assets/wolox-navbar-icon.png")
+    button.logout-button(type="button" @click="logout") Logout
+</template>
+
+<style scoped lang="scss">
+@import '../scss/application';
```

Comment:
As I said in #1 you should only import this in the main component

Reply:
Fixed in another PR